dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -84,8 +84,8 @@ loginFormURL.desc=Specifies the global URL of a form login page including the ro
 loginErrorURL=Login error URL 
 loginErrorURL.desc=Specifies the global URL of a form login error page that includes the root context. The form login error page must be part of a WAR file. An application uses the global login error URL if the application uses form login authentication and does not specify either the form login page or the login error page in the auth-method element of the web.xml file. This is required when overrideHttpAuthMethod attribute is set to FORM.
 
-allowAuthenticationFailOverToAuthMethod=Failover method when certificate login fails
-allowAuthenticationFailOverToAuthMethod.desc=Specifies the authentication fail over method that will be used when the certificate authentication fails or if the certificate is missing. Valid values are BASIC, FORM and APP_DEFINED. APP_DEFINED is only valid when overrideHttpAuthMethod attribute is set to CLIENT_CERT. When APP_DEFINED is set, the authentication method which is  configured in the application is used.
+allowAuthenticationFailOverToAuthMethod=Failover to the authentication method when certificate login fails
+allowAuthenticationFailOverToAuthMethod.desc=Specifies the authentication fail over method that will be used when the certificate authentication fails or if the certificate is missing. Valid values are BASIC, FORM and APP_DEFINED. APP_DEFINED is only valid when overrideHttpAuthMethod attribute is set to CLIENT_CERT. When APP_DEFINED is set, the authentication method which is configured in the application is used.
```

Comment:
- Change "fail over" to "failover"
- Change "will be used" to "is used"
- Add a comma after FORM
- Change "method which" to "method that"

Reply:
Updated in the other PR